.travis.yml

```diff
@@ -13,7 +13,7 @@ matrix:
         - env: PYTHON=3.5 NUMPY=1.11 RUN_COVERAGE=yes CONDA_ENV=travisci
         - env: PYTHON=3.6 NUMPY=1.12 BUILD_DOC=yes CONDA_ENV=travisci
         - env: PYTHON=3.6 NUMPY=1.13 CONDA_ENV=travisci
-        - env: PYTHON=3.6 NUMPY=1.14 CONDA_ENV=travisci
+        - env: PYTHON=3.6 NUMPY=1.14 CONDA_ENV=travisci TEST_SVML=yes
```

Comment:
Do we want to test latest of everything without SVML too?

Reply:
Added test for the config w/o svml